node/package.json

```diff
@@ -4,8 +4,9 @@
     "test": "vtex-test-tools test"
   },
   "dependencies": {
-    "uuid": "^8.3.1",
     "@googlemaps/google-maps-services-js": "^3.1.6",
+    "axios": "0.19.0 || ^0.20.1",
```

Comment:
The version has to be in sync with the google-maps-services-js axios version. Is there a better way to do that? In the future, this might just break.

Reply:
Hmm, that's weird. The Google Maps lib should use axios as a `peerDepedencies` to make this flow work.

Why do you need the `||`? Just the version that works wouldn't solve it?

Reply:
this is the same as `^0.20.1`, so you should probably be able to just leave it as that

Reply:
> Hmm, that's weird. The Google Maps lib should use axios as a peerDepedencies to make this flow work.

Whoops, my bad. Just removed this explicit dependency and it still works.

> Why do you need the ||? Just the version that works wouldn't solve it?

I just [copied from the lib](https://github.com/googlemaps/google-maps-services-js/blob/master/package.json#L47). Good catch